addons/sale/models/sale.py

```diff
@@ -391,6 +391,14 @@ def action_view_invoice(self):
             action = {'type': 'ir.actions.act_window_close'}
         return action
 
+    @api.model
+    def _get_invoice_group_key(self, order):
+        return (order.partner_invoice_id.id, order.currency_id.id)
+
+    @api.model
+    def _get_draft_invoices(self, invoices, references):
```

Comment:
Actually this method does nothing. Adding group key methods as above may be useful but here it is just noise in standard code.

Reply:
@mart-e in the proposed OCA module 'sale_merge_draft_invoice' https://github.com/OCA/sale-workflow/pull/551 we implement the possibility for the user to decide, when an invoice is going to be created, if she wants to merge it with an existing invoice. See how we'd use this method:
https://github.com/OCA/sale-workflow/pull/551/files#diff-a18ef0edee82a9180a943e69bb6f77deR12

We'd be more than happy if this feature was in odoo core! But for now, we just asked for a hook :)

Reply:
So, imagine the scenario where you create an invoice for a sales order for line 1, and then inmediately you create a new invoice, for line 2. You will now get 2 invoices. What we propose is that the user can choose to have 1 single invoice with the 2 lines, if she wants, and as long as the first invoice is still in draft.

Not sure if this would be needed by you guys, or it adds too much complexity to the process. It's up to you to decide :)

Reply:
Thanks for explanations. Sorry but this adds unnecessary complexity for our process.